nose_watcher/nose_watcher.py

```diff
@@ -27,12 +27,16 @@ class WatcherPlugin(Plugin):
     python_files = ('.py', '.pyx')
```

Comment:
I wonder if there was a reason I made this a tuple.
I think if you make this a list, then you wouldn't have to so must casting to tuples and lists